airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -71,7 +71,7 @@
 
 
 JOB_ID_PATTERN = re.compile(
-    r"Submitted job: (?P<job_id_java>[^\"\n]*)|Created job with id: \[(?P<job_id_python>[^\"\n]*)\]"
+    r"Submitted job: (?P<job_id_java>[^\"\n\s]*)|Created job with id: \[(?P<job_id_python>[^\"\n\s]*)\]"
```

Comment:
Looks great! Can we add a unit test for this?

Reply:
I added another test example in job id extraction test

Reply:
thank you!